src/lib/Tab/LocationView/DetailsTab.php

```diff
@@ -152,6 +151,7 @@ public function renderView(array $parameters): string
             'sort_field_clause_map' => $this->getSortFieldClauseMap(),
             'form_state_update' => $contentObjectStateUpdateTypeByGroupId,
             'can_see_section' => $can_see_section,
```

Comment:
Ugh, not your fault but... `$can_see_section` :( Not sure how we overlooked that. Can you fix this?